src/components/TodoList/TodoList.tsx

```diff
@@ -1,6 +1,18 @@
+import classNames from 'classnames';
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
+type Props = {
+  todos: Todo[];
+  selectedTodo?: Todo | null;
```

Comment:
there is no need to add mark `?` because you wrote a default value ` selectedTodo = null` in `TodoList`
so anyway it will be null if we don't pass any todo